libs/feature/editor/src/lib/services/editor.service.ts

```diff
@@ -70,4 +71,8 @@ export class EditorService {
     this.recordsRepository.clearRecordDraft(record.uniqueIdentifier)
     return this.recordsRepository.openRecordForEdition(record.uniqueIdentifier)
   }
+
+  hasRecordChangedSinceDraft(localRecord: CatalogRecord): Observable<string[]> {
+    return this.recordsRepository.hasRecordChangedSinceDraft(localRecord)
+  }
```

Comment:
Not sure if this wrapper is really necessary, but as most methods are called via the service in the effects it might make sense to keep it. I let you judge.